packages/opc-styles/variables/_1-definitions.scss

```diff
@@ -0,0 +1,6 @@
+// global
+// global name for repo
+$repo: opc !default;
+
+// adding support for css custom property by default 
+$USE-FALLBACK: true;
```

Comment:
```suggestion
$USE-FALLBACK: true;

```

Reply:
added the new line at the end of the file.